src/AvPortalResource.php

```diff
@@ -123,6 +123,38 @@ public function getThumbnailUrl(): ?string {
     return NULL;
   }
 
+  /**
+   * Trim text for avoiding maximum string length of varchar.
+   *
+   * @param string $value
+   *   The value which should be trimmed.
+   *
+   * @return string
+   *   The trimmed value.
+   *
+   * @see \Drupal\views\Plugin\views\field\FieldPluginBase::trimText
+   */
+  protected function trimText(string $value): string {
+    // We are using maximum string length of the 'name' column
+    // within the 'media_field_data' table equal to 255 but
+    // with allocated space for the ellipsis symbol.
+    $max_length = 254;
+    if (mb_strlen($value) > $max_length) {
```

Comment:
A string that is long 255 characters shouldn't be cut, but it does now.

Reply:
Currently name field of varchar maximum number of symbols is 255. If after trancate we will have 255 symbols we will not able add ellipsis symbols anymore. because will have again 256 symbols.